lasagne/layers/corrmm.py

```diff
@@ -18,7 +18,7 @@
 ]
 
 
-if not theano.config.device.startswith("gpu"):
+if not theano.config.device.startswith("gpu"):  # pragma: no cover
```

Comment:
You could do something like this:

	def test_import_without_gpu_raises():
	    if theano.config.device.startswith("gpu"):
	        pytest.skip()
	    else:
	        with pytest.raises(ImportError):
	            import lasagne.layers.corrmm